nengo/builder/learning_rules.py

```diff
@@ -86,10 +97,30 @@ def make_step(self, signals, dt, rng):
         theta = signals[self.theta]
         delta = signals[self.delta]
         alpha = self.learning_rate * dt
+        period = (1 if self.apply_every is None else
```

Comment:
Ternary vs. normal is just a preference thing; we use the ternary a lot to cut down on the number of lines. Using an if statement this would be 4 lines versus the 2 it is now. Of course, it can be easier to read 4 short lines than 2 dense lines, so it's a case-by-case thing. I think I prefer ternary here, but if statements would be good too.

As for the refactoring, I could see it being helpful and making the implementation a bit simpler. It would return the same thing, it would just move things around a bit. I can make a commit with that refactoring, or @Seanny123 can (making a commit with the change would be faster than trying to explain the change with words, I think).